ui/raidboss/data/07-dt/ultimate/futures_rewritten.txt

```diff
@@ -65,74 +65,74 @@ hideall "--sync--"
-245.8 "The House of Light" Ability { id: "9D0E", source: "Fatebreaker's Image" }
+244.6 "Axe Kick/Scythe Kick" Ability { id: ["9D0A", "9D0B"], source: "Oracle's Reflection" }
+245.5 "The House of Light" Ability { id: "9D0E", source: ["Fatebreaker's Image", "Oracle's Reflection", "Usurper Of Frost"] }
 247.2 "Frigid Stone" Ability { id: "9D07", source: "Usurper of Frost" }
 247.6 "Icicle Impact" Ability { id: "9D06", source: "Usurper of Frost" }
-248.4 "--sync--" Ability { id: "9CEF", source: "Oracle's Reflection" }
+248.4 "--center--" Ability { id: "9CEF", source: "Oracle's Reflection" }
 251.5 "Heavenly Strike" Ability { id: "9D0F", source: "Usurper of Frost" }
 251.6 "Icicle Impact" Ability { id: "9D06", source: "Usurper of Frost" }
 254.2 "Frigid Needle" Ability { id: "9D08", source: "Usurper of Frost" }
 254.6 "Sinbound Holy (cast)" Ability { id: "9D10", source: "Oracle's Reflection" }
 255.5 "Icicle Impact" Ability { id: "9D06", source: "Usurper of Frost" }
-255.5 "Sinbound Holy" #Ability { id: "9D11", source: "Usurper of Frost" }
+255.5 "Sinbound Holy 1 (puddles)" #Ability { id: "9D11", source: "Usurper of Frost" }
+256.9 "Sinbound Holy 2 (puddles)" #Ability { id: "9D11", source: "Oracle's Reflection" }
+258.5 "Sinbound Holy 3 (puddles)" #Ability { id: "9D11", source: "Oracle's Reflection" }
+260.1 "Sinbound Holy 4 (puddles)" #Ability { id: "9D11", source: "Oracle's Reflection" }
+263.9 "Shining Armor + Frost Armor" Ability { id: ["9FC8", "9CF9"], source: ["Oracle's Reflection", "Usurper Of Frost"] }
+270.5 "Twin Stillness/Twin Silence" Ability { id: ["9D01", "9D02"], source: "Oracle's Reflection" }
+272.6 "Twin Silence/Twin Stillness" Ability { id: ["9D03", "9D04"], source: "Oracle's Reflection" }
+276.2 "--targetable--"
+283.3 "Hallowed Ray" Ability { id: "9D12", source: "Usurper of Frost" }
+293.0 "Mirror, Mirror" Ability { id: "9CF3", source: "Usurper of Frost" }
+307.1 "Scythe Kick" Ability { id: "9D0B", source: "Usurper of Frost" }
+317.2 "Reflected Scythe Kick" Ability { id: "9D0D", source: "Frozen Mirror" }
+323.3 "Banish III" Ability { id: ["9D1C", "9D1D"], source: "Usurper of Frost" }
+326.4 "--center--" Ability { id: "9CEF", source: "Usurper of Frost" }
+332.7 "Light Rampant" Ability { id: "9D14", source: "Usurper of Frost" }
+335.7 "--untargetable--"
+340.7 "Luminous Hammer 1" #Ability { id: "9D1A", source: "Usurper of Frost" }
+342.3 "Luminous Hammer 2" #Ability { id: "9D1A", source: "Usurper of Frost" }
+343.9 "Luminous Hammer 3" #Ability { id: "9D1A", source: "Usurper of Frost" }
+344.0 "Bright Hunger" Ability { id: "9D15", source: "Usurper of Frost" }
+345.4 "Luminous Hammer 4" #Ability { id: "9D1A", source: "Usurper of Frost" }
+347.0 "Luminous Hammer 5" #Ability { id: "9D1A", source: "Usurper of Frost" }
+349.8 "Powerful Light" Ability { id: "9D19", source: "Usurper of Frost" }
+358.8 "Bright Hunger" Ability { id: "9D15", source: "Usurper of Frost" }
```

Comment:
I wonder if it might be worthwhile to have the 344.0 be `Bright Hunger (solo towers)` and the 358.8 be `Bright Hunger (center tower)`, just for clarity?  Feel free to disregard though, they're probably far enough apart to not be confusing.

Also wondering about the whether it would be worth including the `Burst` (9D1B) from the `Holy Light` combatants (the orbs on the ground).  The timing on those is actually probably a Helpful Thing.

Reply:
I suppose it's reasonable enough, yes. Agreed on adding the `Burst` entries, the timing there is useful.